builder/bundles.go

```diff
@@ -480,7 +480,50 @@ func downloadRpms(packagerCmd, rpmList []string, baseDir string, maxRetries int)
 	return nil, downloadErr
 }
 
-func installBundleToFull(packagerCmd []string, buildVersionDir string, bundle *bundle, downloadRetries int) error {
+func extractRpm(baseDir string, rpm string) error {
+	path := baseDir + "/var/cache/yum/clear/"
+	newPath, err := filepath.Glob(filepath.Join(path, "clear-*"))
+	if err != nil {
+		return err
+	}
+	if len(newPath) > 0 {
+		newPath[0] = newPath[0] + "/packages"
+	} else {
+		return fmt.Errorf("cant find rpms packages")
+	}
+
+	rpmFull := rpm + "*.rpm"
+	out, err := filepath.Glob(filepath.Join(newPath[0], rpmFull))
+	if err != nil {
+		return err
+	}
+	if len(out) > 0 {
+		for i := range out {
+			cmd := exec.Command("rpm2cpio", out[i])
```

Comment:
Would you consider adding a check that takes the filelist of the rpm and makes sure that all the paths in it exist already in the chroot before the rpm2cpio is run? This is likely a bit paranoid but I think I would like to have at least a warning available to look at in case something like that comes up that I can track down later in the release process.

Reply:
yup sure.